src/hostapi/pulseaudio/pa_linux_pulseaudio_cb.c

```diff
@@ -825,45 +848,22 @@ PaError PaPulseAudio_StartStreamCb( PaStream * s )
             {
                 PaPulseAudio_Lock( pulseaudioHostApi->mainloop );
 
-                if ( ! pa_stream_connect_playback( stream->outputStream,
+                if ( pa_stream_connect_playback( stream->outputStream,
                                                    pulseaudioName,
                                                    &stream->outputBufferAttr,
                                                    pulseaudioStreamFlags,
                                                    NULL,
                                                    NULL ) )
-                {
-                    pa_stream_set_underflow_callback( stream->outputStream,
-                                                      PaPulseAudio_StreamUnderflowCb,
-                                                      stream);
-                    pa_stream_set_started_callback( stream->outputStream,
-                                                    PaPulseAudio_StreamStartedCb,
-                                                    stream );
-                }
-                else
                 {
                     PA_DEBUG( ("Portaudio %s: Can't write audio!\n",
                               __FUNCTION__) );
```

Comment:
```suggestion
                              __FUNCTION__) );
                       PaPulseAudio_UnLock( pulseaudioHostApi->mainloop );
```
UnLock in error else-case is missing.